azure-pipelines.yml

```diff
@@ -51,6 +46,16 @@ jobs:
   steps:
   - template: scripts/azure-linux_mac.yml
 
+- job: CentOS_manylinux1
+  container:
+    image: quay.io/pypa/manylinux1_x86_64
+  variables:
+    CXX: g++
+    TILEDB_TOOLS: ON
```

Comment:
Let's add serialization here also to validate that on 4.8.5

Reply:
done